### PR TITLE
Move admin behaviour to admin namespace.

### PR DESCRIPTION
**Pull request by mconnell**
Not a terribly exciting PR. We've decided that the admin functionality should be moved into an admin namespace to enforce a separation of concerns.

### CODE REVIEW

**Comment by darokel** [comment]
:+1:

**Comment by Tr4pSt3R** [comment]
:+1: good idea

**Comment by mconnell** [comment]
Thanks! :star2:

File: app/controllers/admin_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AdminController < ApplicationController
+  if Rails.env.production?
+    http_basic_authenticate_with name: ENV['BASIC_AUTH_NAME'], password: ENV['BASIC_AUTH_PASSWORD']
```
Comment (tomas-stefano) [inline app/controllers/admin_controller.rb]: I know that the repo already have the ENV explanation [here](https://github.com/moneyadviceservice/webfeeds#deployment) but I can't see the dot-env or env relate gems in the Gemfile. If I want to use this env variables, do I need to export by hand or do some command? If so, can you add the command setup for that? (or even if is just a copy command). e.g add `cp .env-dev .env` to readme

**Pull request merged**